### PR TITLE
Show number of police and HP during police encounter

### LINKED ISSUE #85: Show number of police and police HP during police encounter
It's currently stored in `life.current`, but we just aren't displaying it.  We should.

### PR DESCRIPTION
**Pull request by jiaming10**
Fix #85

### CODE REVIEW

File: helpers/handlebars.js [hunk 1]
```diff
@@ -212,6 +212,16 @@ hbs.registerHelper("md_partial", function md_partial(partial, opts) {
 	return final;
 });
 
+hbs.registerHelper("get_police_encounter", function log(police) {
+	const startingStr = "As you arrive at the hotel, you see ";
+
+	if (police.officers > 1) {
+		return `${startingStr} ${police.officers} polices arrive behind you. They have ${police.total_hp} HP.`;
```
Comment (snollygolly) [inline helpers/handlebars.js]: It should say "...police officers arrive behind you", or just "...officers arrive behind you", but not polices.

File: helpers/handlebars.js [hunk 2]
```diff
@@ -212,6 +212,16 @@ hbs.registerHelper("md_partial", function md_partial(partial, opts) {
...
+	}
+
+	return `${startingStr} the police arrive behind you. The police has ${police.total_hp} HP.`;
```
Comment (snollygolly) [inline helpers/handlebars.js]: Should say "...have X HP" instead of "...has"

File: helpers/handlebars.js (hunk 1 above)
Comment (jiaming10) [reply helpers/handlebars.js]: I will change all the `police` to `officer` to standardise stuff.

File: helpers/handlebars.js (hunk 2 above)
Comment (jiaming10) [reply helpers/handlebars.js]: If I change to `officer`, should I be using ".....have X HP" or "...has?"

Comment (snollygolly) [reply helpers/handlebars.js]: Yeah, I guess if you change it to "officer", has will be fine.

File: helpers/handlebars.js [hunk 3]
```diff
@@ -212,6 +212,16 @@ hbs.registerHelper("md_partial", function md_partial(partial, opts) {
...
 });
 
+hbs.registerHelper("get_police_encounter", function log(police) {
+	const startingStr = "As you arrive at the hotel, you see ";
+
+	if (police.officers > 1) {
+		return `${startingStr} ${police.officers} officers arrive behind you. They have ${police.total_hp} HP.`;
+	}
+
+	return `${startingStr} the officer arrive behind you. The officer has ${police.total_hp} HP.`;
```
Comment (snollygolly) [inline helpers/handlebars.js]: Sorry, one more small fix.  Now that's it's office, it should be "arrives" not "arrive.

**Comment by snollygolly** [comment]
Fantastic!  Thank you so much for contributing.  I hope you enjoy the game as well.

:shipit:

**Pull request merged**